plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go

```diff
@@ -76,7 +76,7 @@ func calculateBalancedResourceAllocation(pod *v1.Pod, podRequests *schedulercach
 	if glog.V(10) {
 		// We explicitly don't do glog.V(10).Infof() to avoid computing all the parameters if this is
 		// not logged. There is visible performance gain from it.
-		glog.V(10).Infof(
+		glog.Infof(
```

Comment:
Update the comment as well.

Reply:
Why the comment is wrong? ...

Reply:
`We explicitly don't do glog.V(10).Infof()`

The comment seems useless.

Reply:
I am also OK to remove all these comments.
Let's wait for someone else's suggestion.